### DISCUSSION TITLE
能否提供下nvim中adapter的配置，初学者配了好久也没有成功。

### DISCUSSION THREAD

**Discussion by HeraPtt** [actboy168/lua-debug#306] [Q&A]
如题

**Comment by actboy168** [comment]
没有支持nvim。首先需要nvim支持dap，nvim官方不支持dap，虽然有插件支持但特性支持和稳定性都比较差。再次需要一个dap的客户端，这个目前没有插件支持，dap的设计也不如lsp那么通用，所以目前不存在泛用的nvim dap客户端，只能每个调试器自行实现或者适配。

**Reply by HeraPtt** [reply]
> 没有支持nvim。首先需要nvim支持dap，nvim官方不支持dap，虽然有插件支持但特性支持和稳定性都比较差。再次需要一个dap的客户端，这个目前没有插件支持，dap的设计也不如lsp那么通用，所以目前不存在泛用的nvim dap客户端，只能每个调试器自行实现或者适配。

好的。谢谢
